networks/suzuka/suzuka-full-node/src/da_db.rs

```diff
@@ -32,7 +32,7 @@ impl DaDB {
 		Ok(Self { inner: Arc::new(db) })
 	}
 
-	pub async fn add_executed_block(&self, id: String) -> Result<(), anyhow::Error> {
+	pub async fn add_executed_block(&self, id: Vec<u8>) -> Result<(), anyhow::Error> {
```

Comment:
Should we just use an `Id` domain type?

Reply:
Yeah, we should; was going to leave some comments about that.

Reply:
I see, the thing is we don't have domain types to represent these values yet, so we just operate with the raw types that prost-build gives us.

But since da_db is an internal module of suzuka-full-node at this point, it's not that important.